### PR TITLE
macs2-callpeak Command Line Tool definition

### PR DESCRIPTION
**Pull request by alexbarrera**
MACS2 callpeak subcommand CWL definition. Compared with the original MACS2 callpeak command, the only difference is the usage of the first treatment sample as basename for the output files produced (-n parameter).

### CODE REVIEW

File: tools/macs2-callpeak.cwl [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+# ------ Input files arguments ----- #
+# ---------------------------------- #
+  - id: t
+    type:
+      type: array
+      items: File
+    description: "Treatment sample file(s). If multiple files are given as -t A B C, then they will all be read and pooled together. IMPORTANT: the first sample will be used as the outputs basename."
+    inputBinding:
+      position: 2
+      prefix: -t
```
Comment (mr-c) [inline tools/macs2-callpeak.cwl]: Is there a "long" form of this option, or is `-t` the only way?

**Comment by mr-c** [comment]
Thank you @alexbarrera for your contribution. I left a question in-line about the availability of long-form options.

**Comment by alexbarrera** [comment]
Hi @mr-c. There are --treatment and --control instead of -t and -c. Are long forms preferred over short ones?

**Comment by mr-c** [comment]
From a readability and debugging perspective I recommend them. We don't
(yet) have a formal style guide.

On Mon, May 9, 2016 at 6:12 PM, Alejandro Barrera <notifications@github.com>
wrote:

> Hi @mr-c <https://github.com/mr-c>. There are --treatment and --control
> instead of -t and -c. Are long forms preferred over short ones?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/common-workflow-language/workflows/pull/81#issuecomment-217911035>
>



-- 
Michael R. Crusoe                          michael.crusoe@gmail.com
Community Engineer                 Common Workflow Language project
https://impactstory.org/u/0000-0002-2961-9670   +32 (0) 2 808 25 58
                                                    +1 480 627 9108

**Comment by portah** [comment]
I like this CWL description, we are going to use it :) soon. 
@mr-c the only question that I have is about docker, I'm okey with ubuntu's package. However, we already use the latest github version. How we can deal with this in the future?

**Comment by mr-c** [comment]
@portah has the CLI interface changed between the two versions? You can always backport a newer package to any Debian or Ubuntu supported release

**Comment by mr-c** [comment]
Thank you @alexbarrera !

**Comment by alexbarrera** [comment]
@mr-c no problem!

**Pull request merged**